cmd/eksctl-anywhere/cmd/upgradecluster.go

```diff
@@ -24,6 +24,7 @@ type upgradeClusterOptions struct {
 	forceClean            bool
 	hardwareCSVPath       string
 	tinkerbellBootstrapIP string
+	e2eTimeout            bool
```

Comment:
Why do we need to expose this as an `e2e-timeout`? Ideally we wouldn't have any flag that refers to something as e2e, but if this is something that a user can see, can we call it `external-etcd-timeout`, or have an overridable timeout for all waits like `node-wait-timeout`?

Reply:
If we don't want to make that decision now, I would maybe opt in for this to be an envvar configuration rather than a flag.